pan_firewall/setup.py

```diff
@@ -0,0 +1,64 @@
+# (C) Datadog, Inc. 2021-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+from codecs import open  # To use a consistent encoding
```

Comment:
This is a tile-only integration, why are you adding this file?

Reply:
Thanks for letting me know. 

I added it because of job failure https://dev.azure.com/datadoghq/integrations-core/_build/results?buildId=71086&view=logs&j=910f82a4-622d-5ace-f877-8b75c045486c&t=66e8cc47-ff40-5964-82bf-b24377b9a6cd

To confirm, since this is a tile only integration, can I remove the following files/folders?
- this setup.py to be reverted
- configuration dir and spec - [pan_firewall/assets/configuration/](https://github.com/DataDog/integrations-core/tree/master/pan_firewall/assets/configuration)
- datadog_checks dir - [pan_firewall/datadog_checks/](https://github.com/DataDog/integrations-core/tree/master/pan_firewall/datadog_checks)

Reply:
It seems it is actually a logs-only integration https://github.com/DataDog/integrations-core/pull/8637

`ddev create -n` resulted in the following

```
pan_firewall
├── assets
│   └── configuration
│       └── spec.yaml
│   └── dashboards
│       └── pan_firewall_overview.json
│   └── service_checks.json
├── datadog_checks
│   └── pan_firewall
│       └── data
│           └── conf.yaml.example
│       ├── __about__.py
│       └── __init__.py
│   └── __init__.py
├── images
│   └── .gitkeep
├── CHANGELOG.md
├── MANIFEST.in
├── README.md
├── manifest.json
├── metadata.csv
├── requirements-dev.txt
├── requirements.in
└── setup.py
```